protob/messages.proto

```diff
@@ -913,6 +922,57 @@ message CosiSignature {
 	optional bytes signature = 1;		// Signature
 }
 
+//////////////////////
+// Stellar messages //
+//////////////////////
+
+/**
+ * In: request for the public key at the specified index
+ */
+message StellarGetPublicKey {
+	required uint32 index = 1;			// Account index (derived as m/44'/148'/index')
+}
+
+/**
+ * Out: public key for the given index
+ */
+message StellarPublicKey {
+	required uint32 index = 1;			// Account index
+	required bytes public_key = 2;		// Raw bytes of the public key (no version or checksum)
+}
+
+/**
+ * In: request to sign the given transaction
+ */
+message StellarSignTx {
+	required uint32 index = 1;					// Account index (derived as m/44'/148'/index')
```

Comment:
Is there any specific reason to not use `repeated uint32 address_n`?

Reply:
I worked with the Stellar team and the developer working on Ledger support to standardize on a specific derivation path where the only thing the user can choose is an index. To keep things consistent, I'd prefer to prevent the user from overriding this.

More details here: https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0005.md